### ISSUE TITLE
Links to non existing articles

### ISSUE THREAD

**Issue by NijazM** [gnosygnu/xowa#789]
Can you somehow make links to nonexisting articles different than from existing articles? For example remove them or make them different color?
Here are some, and page on which they are appearing:

`en.wikipedia.org/wiki/Scattered_and/or_isolated_thunderstorms`
appearing on `en.wikipedia.org/wiki/Brontophobia`

`en.wikipedia.org/wiki/Stockade_fence`
appearing on `en.wikipedia.org/wiki/Fence`

**Comment by desb42** [comment]
I suspect that these pages are being requested using xowa-http (as xowa-gui does produce redlinks)
if so, this issue is indirectly noted in #272

**Comment by NijazM** [comment]
Yes, just noticed. Only in server. Hope you can do something about it.

**Comment by catmaps** [comment]
I like redlinks for testing but prefer black text regular regular reading/study. To do black, go to Tools>Options>HTML-CSS. In the custom CSS box paste this:
.new {color:inherit !important;}
Screenshot:
![image](https://user-images.githubusercontent.com/73363722/102223550-dbf06e00-3eb2-11eb-90c0-1fba1ff4bb43.png)

Gnosygnu has made this a very nice customizable program. (For earlier versions of Xowa I used .new {color:initial !important;}
